### PR TITLE
Update godoc for discovery registration

### PR DESCRIPTION
**Pull request by liggitt**
Follow up from https://github.com/kubernetes/kubernetes/pull/43553/files?diff=split#r107723539

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go [hunk 1]
```diff
@@ -331,6 +331,8 @@ func (s *GenericAPIServer) InstallAPIGroup(apiGroupInfo *APIGroupInfo) error {
 	return nil
 }
 
+// AddAPIGroupForDiscovery adds the specified group to the list served to discovery queries.
+// Discovery lists groups in order of registration.
```
Comment (cblecker) [inline staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go]: Is registration the right term here? I don't see it used else where, and perhaps something like "... in the order in which they were added" may be clearer.

Comment (liggitt) [inline staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go]: updated

**Comment by cblecker** [comment]
This LGTM, but I'll let @eparis give the final thumbs up before merge.

**Pull request merged**